todo/src/actions/index.js

```diff
@@ -0,0 +1,23 @@
+export const GET_TODOS = "GET_TODOS";
+export const ADD_TODO = "ADD_TODO";
+export const COMPLETED_TODO = "COMPLETED_TODO";
+
+export const todos = () => {
+  return {
+    type: GET_TODOS
+  };
+};
+
+export const addTodo = todo => {
```

Comment:
```suggestion
export const addTodo = todo => ({
    type: ADD_TODO,
    payload: todo
})
```

Reply:
You can extract a little boilerplate when using these es6 arrow functions with implicit returning.